### ISSUE TITLE
Published app fails to run when referencing package with satellite assemblies and targeting netcoreapp2.0

### ISSUE THREAD

**Issue by pakrym** [dotnet/core-setup#2157]
## Steps to reproduce

1. dotnet new web 
2. Change package reference from Microsoft.AspNetCore to Microsoft.AspNetCore.All  
3. dotnet restore
3. dotnet publish 
4. dotnet repoapp.dll

## Expected  behavior
App runs

## Actual behavior
`Error: assembly specified in the dependencies manifest was not found -- package: 'Microsoft.Data.Edm', version: '5.8.2', path: 'lib/netstandard1.1/de/Microsoft.Data.Edm.resources.dll'`

## Environment data

```
.NET Command Line Tools (2.0.0-preview1-005825)

Product Information:
 Version:            2.0.0-preview1-005825
 Commit SHA-1 hash:  b4a821109d

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Users\pakrym\AppData\Local\Microsoft\dotnet\sdk\2.0.0-preview1-005825\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0-preview1-002061-00
  Build    : 2b70ec9c3b014af0c2a5f45de0e5b73a1ae51c09

```

**Comment by eerhardt** [comment]
This is a regression from `netcoreapp1.1` to `netcoreapp2.0`.

A simpler/smaller repro:

```XML
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp1.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="System.Spatial" Version="*" />
  </ItemGroup>

</Project>
```

Using the latest 2.0 SDK and runtime, restore, publish and run that project.  It runs fine.

Now change  `<TargetFramework>netcoreapp1.1</TargetFramework>` to     `<TargetFramework>netcoreapp2.0</TargetFramework>`.

Restore, publish and run that app, and it fails with the same error message.

It appears to me to be a regression caused by https://github.com/dotnet/core-setup/pull/1392.

**Comment by gkhanna79** [comment]
Fix in PR https://github.com/dotnet/core-setup/pull/2160

**Comment by gkhanna79** [comment]
Fix merged in master and ported to release/2.0.0 as https://github.com/dotnet/core-setup/pull/2164

**Comment by gkhanna79** [comment]
release/2.0.0 PR is merged and fix is in Preview1.
